packages/backend/server/src/modules/workspaces/permission.ts

```diff
@@ -21,6 +21,25 @@ export class PermissionService {
     return data?.type as Permission;
   }
 
+  async getOwnedWorkspaces(userId: string) {
+    return this.prisma.userWorkspacePermission
+      .findMany({
+        where: {
+          userId,
+          accepted: true,
+          type: Permission.Owner,
+        },
+        select: {
+          workspace: {
```

Comment:
there is workspace id in permission table, no need to join query if only id needed